mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -1033,6 +1033,16 @@
 									"        pm.environment.set(\"custom-package-id-created-in-post\", response.data.id);",
 									"    }",
 									"    ",
+									"//Check that tagList is not empty",
+									"pm.test('TagList is not empty', function(){",
+									"    pm.expect(response.data.attributes.tags.tagList.size) !== 0;",
+									"});",
+									"    ",
+									"//Check that tagList is not empty",
+									"pm.test('Creating tagList', function(){",
+									"    pm.expect(response.data.attributes.tags.tagList[0]).eq(\"foo-tag\");",
+									"});",
+									"    ",
```

Comment:
Below this -- where we check for expected attributes (line 1054/1055) -- should we also check for tags to be complete?
      pm.expect(response.data.attributes).to.include.all.keys("contentType", "customCoverage", "isCustom", "isSelected", "name", "packageId", 
        "packageType", "providerId", "providerName", "selectedCount", "titleCount", "visibilityData", "allowKbToAddTitles");